sw-precache.js

```diff
@@ -119,11 +120,13 @@ module.exports = function(params) {
   params.logger(util.format('Total precache size is about %s for %d resources.',
     formatBytesAsString(cumulativeSize), relativeUrls.length));
 
-  var templateBuffer = fs.readFileSync(params.templateFilePath);
-  return _.template(templateBuffer, {
+  var template = fs.readFileSync(params.templateFilePath, {encoding: 'utf8'});
+  var populatedTemplate = _.template(template, {
     handleFetch: params.handleFetch,
     importScripts: params.importScripts ? params.importScripts.map(JSON.stringify).join(',') : null,
     includeCachePolyfill: params.includeCachePolyfill,
     precacheConfig: JSON.stringify(precacheConfig)
   });
+
+  return resumer().queue(populatedTemplate).end();
```

Comment:
@sindresorhus I suggested returning a stream here to make it cleaner to pipe through the output to other tasks. A callback based API would work too. I just felt a stream would work better.

Btw, quick note on resumer here: That module is great and offers you a one-liner, but the auto-resumption of the stream on nextTick can nip you in the butt if you expect to pass the stream to an unknown consumer.

@sindresorhus Happy to defer to your best judgement on whether streams should be used here.